src/app/ipv/middleware.js

```diff
@@ -238,9 +238,9 @@ module.exports = {
         return res.render(`ipv/${req.session.currentPage}.njk`, {
           context: "unrecoverable",
         });
-      } else if (pageId === "pyi-timeout-unrecoverable") {
```

Comment:
As discussed, we probably need to leave this section in temporarily until the back end is updated to use the new target.